assets/common/common.go

```diff
@@ -51,6 +51,13 @@ func HandleFatalError(err error, msg string) {
 	os.Exit(1)
 }
 
+func HandleNonFatalError(err error, msg string) {
```

Comment:
Did you intend to use this new function somewhere? I don't see that it is used...

Reply:
good catch, fixed!